backend/data/migrations/20200321015501_update_users_table.js

```diff
@@ -0,0 +1,21 @@
+module.exports.up = function(knex) {
+  return knex.schema.table("users", users => {
+    users.string("blurb", 255);
+    users.string("country", 255);
+    users.string("city", 255);
+    users.boolean("email_verification").defaultTo(false);
+    users.boolean("phone_verification").defaultTo(false);
```

Comment:
I don't think that's possible since user does not have a column for a phone. Actually I think email verification is not possible neither since it requires some email preconfiguration.